tests/unit/models/transactions/test_permissioned_domain_delete.py

```diff
@@ -37,7 +42,9 @@ def test_domain_id_too_long(self):
 
         self.assertEqual(
             err.exception.args[0],
-            "{'PermissionedDomainDelete': 'domain_id must be 64 characters long.'}",
+            "{'PermissionedDomainDelete': 'domain_id must be "
+            + str(DOMAIN_ID_LENGTH)
+            + " characters long.'}",
```

Comment:
ditto

Reply:
how is this better than what I already have in the code?

Reply:
[8206d73](https://github.com/XRPLF/xrpl-py/pull/773/commits/8206d73d87680dd34b3a008857a439eece88405d)